core/templates/dev/head/components/ExplorationSummaryTileDirective.js

```diff
@@ -78,6 +79,13 @@ oppia.directive('explorationSummaryTile', [function() {
           }
         );
 
+        if ($scope.isCommunityOwned) {
+          $scope.contributors.unshift('Community Owned');
+          $scope.numMinusOneText = $scope.contributors.length - 1;
+        } else {
+          $scope.numMinusOneText = '+' + $scope.contributors.length - 1;
```

Comment:
Might be cleaner to do an ng-if in the html (around the '+' sign) based on isCommunityOwned().